src/Okta.Sdk.UnitTests/UsersClientShould.cs

```diff
@@ -124,5 +125,464 @@ public async Task RemoveApplicationTargetFromApplicationAdministratorRoleForUser
+                                        ""issuer"": ""https://${yourOktaDomain}/oauth2/ausain6z9zIedDCxB0h7"",
+                                        ""clientId"": ""0oabskvc6442nkvQO0h7"",
+                                        ""userId"": ""00u5t60iloOHN9pBi0h7"",
+                                        ""scopeId"": ""scpCmCCV1DpxVkCaye2X"",
+                                        ""_links"": {
+                                            ""app"": {
+                                                ""href"": ""https://${yourOktaDomain}/api/v1/apps/0oabskvc6442nkvQO0h7"",
+                                                ""title"": ""My App""
+                                            },
+                                            ""scope"": {
+                                                ""href"": ""https://${yourOktaDomain}/api/v1/authorizationServers/ausain6z9zIedDCxB0h7/scopes/scpCmCCV1DpxVkCaye2X"",
+                                                ""title"": ""My phone""
+                                            },
+                                            ""client"": {
+                                                ""href"": ""https://${yourOktaDomain}/oauth2/v1/clients/0oabskvc6442nkvQO0h7"",
+                                                ""title"": ""My App""
+                                            },
+                                            ""self"": {
+                                                ""href"": ""https://${yourOktaDomain}/api/v1/users/00u5t60iloOHN9pBi0h7/grants/oag3ih1zrm1cBFOiq0h6"",
+                                                ""hints"": {
+                                                    ""allow"": [
+                                                        ""GET"",
+                                                        ""DELETE""
+                                                    ]
+                                                }
+                                            },
+                                            ""user"": {
+                                                ""href"": ""https://${yourOktaDomain}/api/v1/users/00u5t60iloOHN9pBi0h7"",
+                                                ""title"": ""SAML Jackson""
+                                            },
+                                            ""authorizationServer"": {
+                                                ""href"": ""https://${yourOktaDomain}/api/v1/authorizationServers/ausain6z9zIedDCxB0h7"",
+                                                ""title"": ""Example Authorization Server""
+                                            }
+                                        }
+                                    }
+                                ]";
+            var mockRequestExecutor = new MockedStringRequestExecutor(rawResponse);
+            var client = new TestableOktaClient(mockRequestExecutor);
+            await client.Users.ListUserGrants("foo").ToListAsync();
```

Comment:
I suggest verifying that the response is parsed properly, and it has the object we are expecting.